src/browser/ui/ReactMount.js

```diff
@@ -451,12 +457,13 @@ var ReactMount = {
    * perform an update on it and only mutate the DOM as necessary to reflect the
    * latest React component.
    *
+   * @param {ReactElement} parentComponent The conceptual parent of this render tree.
    * @param {ReactElement} nextElement Component element to render.
    * @param {DOMElement} container DOM element to render into.
    * @param {?function} callback function triggered on completion
    * @return {ReactComponent} Component instance rendered in `container`.
    */
-  render: function(nextElement, container, callback) {
+  renderSubtreeIntoContainer: function(parentComponent, nextElement, container, callback) {
```

Comment:
I don't like the order of arguments here. If we're going to make this public we probably want some additional error messages (eg, if I call this method as a user without a `parentComponent` then we should hit an invariant).

If we make this an instance method it could have the same signature as `React.render`...